msf_updates/include/msf_updates/pressure_sensor_handler/implementation/pressure_sensorhandler.hpp

```diff
@@ -30,8 +30,8 @@ PressureSensorHandler::PressureSensorHandler(
   ros::NodeHandle pnh("~/pressure_sensor");
   ros::NodeHandle nh("msf_updates");
   subPressure_ =
-      nh.subscribe<asctec_hl_comm::mav_imu>
-      ("pressure_input", 20, &PressureSensorHandler::MeasurementCallback, this);
+      nh.subscribe<geometry_msgs::PointStamped>
+      ("pressure_height", 20, &PressureSensorHandler::MeasurementCallback, this);
```

Comment:
I'd say change it in the launchfile, then we can merge this. We still need to implement it in asctec_mav_framework